packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -988,11 +994,25 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
       selectionGeometry: _selectionDelegate.value,
       onCopy: () {
         _copy();
-        hideToolbar();
+
+        // In Android copy should clear the selection.
+        switch (defaultTargetPlatform) {
+          case TargetPlatform.android:
+            _clearSelection();
+            break;
+          case TargetPlatform.iOS:
+            hideToolbar(false);
+            break;
+          case TargetPlatform.fuchsia:
+          case TargetPlatform.linux:
+          case TargetPlatform.macOS:
+          case TargetPlatform.windows:
+            hideToolbar();
+            break;
+        }
       },
       onSelectAll: () {
-        selectAll();
-        hideToolbar();
+        selectAll(SelectionChangedCause.toolbar);
```

Comment:
I think this also might depend on the platform.  With a quick test:

| Platform | Does it hide the toolbar after select all? |
| --- | --- |
| MacOS | Yes (on Iterm. I also tried Chrome and TextEdit, which had no "Select all" button) |
| Linux | Yes (on gEdit.  Chrome and Terminal didn't have "Select all") |
| Android | No |
| iOS | No |

So probably desktop platforms should hide the toolbar and mobile platforms should not?

Reply:
thank you for the informations, I fixed it~